ethereum/core/src/main/java/org/hyperledger/besu/ethereum/bonsai/worldview/BonsaiWorldStateUpdateAccumulator.java

```diff
@@ -328,17 +328,6 @@ public void commit() {
                 }
                 if (tracked.getStorageWasCleared()) {
                   updatedAccount.clearStorage();
-                  wrappedWorldView()
-                      .getAllAccountStorage(updatedAddress, updatedAccount.getStorageRoot())
-                      .forEach(
-                          (keyHash, entryValue) -> {
-                            final StorageSlotKey storageSlotKey =
-                                new StorageSlotKey(Hash.wrap(keyHash), Optional.empty());
-                            final UInt256 value = UInt256.fromBytes(RLP.decodeOne(entryValue));
-                            pendingStorageUpdates.put(
-                                storageSlotKey, new BonsaiValue<>(value, null, true));
-                          });
-                  updatedAccount.setStorageRoot(Hash.EMPTY_TRIE_HASH);
```

Comment:
@matthew1001 is this change related?

Reply:
I don't know what happened with my branch but this commit has now been removed.